example/pubspec.yaml

```diff
@@ -11,14 +11,18 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_map:
-    path: ../
+  flutter_map_cancellable_tile_provider:
   latlong2: ^0.9.0
   proj4dart: ^2.1.0
   url_launcher: ^6.1.10
   shared_preferences: ^2.1.1
   url_strategy: ^0.2.0
   http: ^1.1.0
-  dio: ^5.3.2
+
+dependency_overrides:
+  flutter_map_cancellable_tile_provider:
```

Comment:
Does it really work to use `flutter_map_cancellable_tile_provider` without any version or path? 🤔